pkg/status/status_otlp.go

```diff
@@ -8,24 +8,39 @@
 package status
 
 import (
-	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"fmt"
+	"io"
+	"net/http"
+
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/otlp"
 )
 
 // GetOTLPStatus parses the otlp pipeline and its collector info to be sent to the frontend
 func GetOTLPStatus() map[string]interface{} {
-	status := make(map[string]interface{})
-	otlpIsEnabled := otlp.IsEnabled(config.Datadog)
-	var otlpCollectorStatus otlp.CollectorStatus
-	if otlpIsEnabled {
-		otlpCollectorStatus = otlp.GetCollectorStatus(common.OTLP)
-	} else {
-		otlpCollectorStatus = otlp.CollectorStatus{Status: "Not running", ErrorMessage: ""}
+	if !otlp.IsEnabled(config.Datadog) {
+		return map[string]interface{}{
+			"otlpStatus":          false,
+			"otlpCollectorStatus": otlp.CollectorStatus{Status: "Not running"},
+		}
 	}
 
-	status["otlpStatus"] = otlpIsEnabled
-	status["otlpCollectorStatus"] = otlpCollectorStatus.Status
-	status["otlpCollectorStatusErr"] = otlpCollectorStatus.ErrorMessage
-	return status
+	var status, statuserr string
+	resp, err := getHTTPClient().Get("http://localhost:13133")
```

Comment:
We should make the port configurable so that people can change it to avoid collisions. One instance where collisions may happen frequently is if the user is also running a Collector with the healthcheck extension enabled on the same host. We may want to change the port used to avoid that collision.

Reply:
Indeed, I wanted to read it from config as well. But this component isn't componetised, so I'm not sure how that'll work. I'll try looking into it more deeply.

Reply:
Done